[2020-12-01T16:46:44.604] [ERROR] orders - ReferenceError: Query is not defined
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:17:23
    at Layer.handle [as handle_request] (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\index.js:47:12)
[2020-12-02T21:43:07.988] [ERROR] orders - Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2nd Stage, Indiranagar, Bengaluru, Karnataka 560038, "Digital",[object Object] )' at line 2
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:62:38
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: `You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2nd Stage, Indiranagar, Bengaluru, Karnataka 560038, "Digital",[object Object] )' at line 2`,
  sqlState: '42000',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    (Product_ID, Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES \n' +
    '    (33,1,HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038, "Digital",[object Object] ) '
}
[2020-12-02T21:43:27.159] [ERROR] orders - Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2nd Stage Indiranagar Bengaluru Karnataka 560038, "Digital",[object Object],[obj' at line 2
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:62:38
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: `You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '2nd Stage Indiranagar Bengaluru Karnataka 560038, "Digital",[object Object],[obj' at line 2`,
  sqlState: '42000',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    (Product_ID, Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES \n' +
    '    (33,1,HAL 2nd Stage Indiranagar Bengaluru Karnataka 560038, "Digital",[object Object],[object Object] ) '
}
[2020-12-02T21:43:50.086] [ERROR] orders - Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object],[object Object],[object Object] )' at line 2
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:62:38
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object],[object Object],[object Object] )' at line 2",
  sqlState: '42000',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    (Product_ID, Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES \n' +
    '    (33,1,"HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038", "Digital",[object Object],[object Object],[object Object] ) '
}
[2020-12-02T21:44:18.956] [ERROR] orders - Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object],[object Object],[object Object],[object Object] )' at line 2
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:62:38
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object],[object Object],[object Object],[object Object] )' at line 2",
  sqlState: '42000',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    (Product_ID, Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES \n' +
    '    (33,1,"HAL 2nd Stage Indiranagar Bengaluru Karnataka 560038", "Digital",[object Object],[object Object],[object Object],[object Object] ) '
}
[2020-12-02T21:45:01.933] [ERROR] orders - Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object],[object Object],[object Object],[object Object],[object Object] ' at line 2
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:64:38
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object],[object Object],[object Object],[object Object],[object Object] ' at line 2",
  sqlState: '42000',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    (Product_ID, Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES \n' +
    '    (33,1,"HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038", "Digital",[object Object],[object Object],[object Object],[object Object],[object Object] ) '
}
[2020-12-02T21:45:46.044] [ERROR] orders - Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object],[object Object],[object Object],[object Object],[object Object],' at line 2
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:64:38
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object],[object Object],[object Object],[object Object],[object Object],' at line 2",
  sqlState: '42000',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    (Product_ID, Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES \n' +
    '    (33,1,"HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038", "Digital",[object Object],[object Object],[object Object],[object Object],[object Object],[object Object] ) '
}
[2020-12-02T21:46:23.289] [ERROR] orders - Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object],[object Object],[object Object],[object Object],[object Object],' at line 2
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:64:38
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object],[object Object],[object Object],[object Object],[object Object],' at line 2",
  sqlState: '42000',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    (Product_ID, Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES \n' +
    '    (33,1,"HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038", "Digital",[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object] ) '
}
[2020-12-02T22:00:44.951] [ERROR] orders - Error: ER_TRUNCATED_WRONG_VALUE: Incorrect date value: '2006' for column 'Date' at row 1
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:48:11
    at Layer.handle [as handle_request] (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\layer.js:95:5) {
  code: 'ER_TRUNCATED_WRONG_VALUE',
  errno: 1292,
  sqlMessage: "Incorrect date value: '2006' for column 'Date' at row 1",
  sqlState: '22007',
  index: 0,
  sql: 'INSERT INTO orders\n' +
    '    (Customer_ID,Date,Price,Product_ID,quantity)\n' +
    '    VALUES (1, 2020-12-02, 3000, 33, 3) '
}
[2020-12-02T22:04:46.771] [ERROR] orders - Error: ER_NO_REFERENCED_ROW_2: Cannot add or update a child row: a foreign key constraint fails (`shoppy`.`orders`, CONSTRAINT `orders_ibfk_2` FOREIGN KEY (`Product_ID`) REFERENCES `product` (`Product_ID`))
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:48:11
    at Layer.handle [as handle_request] (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\layer.js:95:5) {
  code: 'ER_NO_REFERENCED_ROW_2',
  errno: 1452,
  sqlMessage: 'Cannot add or update a child row: a foreign key constraint fails (`shoppy`.`orders`, CONSTRAINT `orders_ibfk_2` FOREIGN KEY (`Product_ID`) REFERENCES `product` (`Product_ID`))',
  sqlState: '23000',
  index: 0,
  sql: 'INSERT INTO orders\n' +
    '    (Customer_ID,Date,Price,Product_ID,quantity)\n' +
    '    VALUES (1, "2020-12-02", 3800, 38, 3) '
}
[2020-12-04T15:15:04.087] [ERROR] orders - Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:58:11
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlMessage: "Unknown column 'undefined' in 'field list'",
  sqlState: '42S22',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    (Product_ID, Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES\n' +
    '    (33,1,"HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038", "Digital",undefined) '
}
[2020-12-04T15:15:50.879] [ERROR] orders - Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:58:11
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlMessage: "Unknown column 'undefined' in 'field list'",
  sqlState: '42S22',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    (Product_ID, Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES\n' +
    '    (33,1,"HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038", "Digital",undefined) '
}
[2020-12-04T15:16:25.427] [ERROR] orders - Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:58:11
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlMessage: "Unknown column 'undefined' in 'field list'",
  sqlState: '42S22',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    (Product_ID, Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES\n' +
    '    (33,1,"HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038", "Digital",undefined) '
}
[2020-12-04T15:46:05.175] [ERROR] orders - Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:58:11
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlMessage: "Unknown column 'undefined' in 'field list'",
  sqlState: '42S22',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    (Product_ID, Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES\n' +
    '    (33,1,"HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038", "Digital",undefined) '
}
[2020-12-04T15:46:18.705] [ERROR] orders - Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:58:11
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlMessage: "Unknown column 'undefined' in 'field list'",
  sqlState: '42S22',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    (Product_ID, Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES\n' +
    '    (33,1,"HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038", "Digital",undefined) '
}
[2020-12-04T16:37:25.306] [ERROR] orders - Error: ER_BAD_FIELD_ERROR: Unknown column 'Price' in 'field list'
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:54:13
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlMessage: "Unknown column 'Price' in 'field list'",
  sqlState: '42S22',
  index: 0,
  sql: 'INSERT INTO orders\n' +
    '    (Order_id,Customer_ID,Date,Price,quantity)\n' +
    '    VALUES ( null, 1, "2020-12-04", 3800, 38)'
}
[2020-12-04T16:39:58.006] [ERROR] orders - ReferenceError: productID is not defined
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:64:8
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2020-12-04T16:41:46.985] [ERROR] orders - Error: ER_BAD_FIELD_ERROR: Unknown column 'Paymode' in 'field list'
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:61:13
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlMessage: "Unknown column 'Paymode' in 'field list'",
  sqlState: '42S22',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '    ( Customer_ID, Address, Paymode, Order_id)\n' +
    '    VALUES\n' +
    '    (1,"HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038", "Digital",null) '
}
[2020-12-04T16:55:16.363] [ERROR] orders - Error: ER_DUP_ENTRY: Duplicate entry '1' for key 'PRIMARY'
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:56:13
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: "Duplicate entry '1' for key 'PRIMARY'",
  sqlState: '23000',
  index: 0,
  sql: 'INSERT INTO orders\n' +
    '      (Order_id,Customer_ID,Date,Price,quantity,Product_ID)\n' +
    '      VALUES ( 1, 1, "2020-12-04", 3800, 38,38)'
}
[2020-12-04T17:21:34.563] [ERROR] orders - Error: ER_NO_SUCH_TABLE: Table 'shoppy.shipping' doesn't exist
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:63:13
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_NO_SUCH_TABLE',
  errno: 1146,
  sqlMessage: "Table 'shoppy.shipping' doesn't exist",
  sqlState: '42S02',
  index: 0,
  sql: 'INSERT INTO shipping\n' +
    '      ( Customer_ID, Address, Paymode, Order_id)\n' +
    '      VALUES\n' +
    '      (1,"HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038", "Digital",1) '
}
[2020-12-04T17:31:22.164] [ERROR] orders - Error: ER_DATA_TOO_LONG: Data too long for column 'address' at row 1
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:59:13
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'ER_DATA_TOO_LONG',
  errno: 1406,
  sqlMessage: "Data too long for column 'address' at row 1",
  sqlState: '22001',
  index: 0,
  sql: 'INSERT INTO orders\n' +
    '      (\n' +
    '      Customer_ID,\n' +
    '      Quantity,\n' +
    '      Product_ID,\n' +
    '      Price,\n' +
    '      current_order_id,\n' +
    '      address,\n' +
    '      shippind_date\n' +
    '      )\n' +
    '      VALUES (\n' +
    '        1, \n' +
    '        8, \n' +
    '        42, \n' +
    '        3800,\n' +
    '        6,\n' +
    '        "Kemapura Dakale (Agrahara), XXI division, Bangalore city corporation, Nagarbhavi Main road, Govindaraja Nagar Ward, Prashant Nagar, Bengaluru, Karnataka 560079", \n' +
    '        "2020-12-04"\n' +
    '      )'
}
[2020-12-04T19:48:12.488] [ERROR] orders - ReferenceError: req is not defined
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:23:29
    at Layer.handle [as handle_request] (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\index.js:275:10)
[2020-12-04T19:48:27.849] [ERROR] orders - Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'on clause'
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:25:29
    at Layer.handle [as handle_request] (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\layer.js:95:5) {
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlMessage: "Unknown column 'undefined' in 'on clause'",
  sqlState: '42S22',
  index: 0,
  sql: '\n' +
    '    SELECT product.product_image1      AS image1, \n' +
    '       product.product_image2      AS image1, \n' +
    '       product.product_name        AS productName, \n' +
    '       product.product_description AS productDescription, \n' +
    '       orders.price                AS price, \n' +
    '       orders.shippind_date        AS shippingDate, \n' +
    '       orders.shipping_status      AS shippingStatus, \n' +
    '       orders.address              AS shippingAddress, \n' +
    '       orders.quantity             AS quantity \n' +
    'FROM   product \n' +
    '       INNER JOIN orders \n' +
    '               ON orders.customer_id = undefined\n' +
    '    '
}
[2020-12-04T19:50:03.590] [ERROR] orders - Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'on clause'
    at Query.Sequence._packetToError (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Protocol._enqueue (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:6:21
    at new Promise (<anonymous>)
    at promisifiedQueryWrapper (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\database\queryInstance.js:5:10)
    at C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\Api\Orders\Orders.api.js:25:29
    at Layer.handle [as handle_request] (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\pavan\Desktop\stuffs\Shoppy\server\node_server\node_modules\express\lib\router\layer.js:95:5) {
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlMessage: "Unknown column 'undefined' in 'on clause'",
  sqlState: '42S22',
  index: 0,
  sql: '\n' +
    '    SELECT product.product_image1      AS image1, \n' +
    '       product.product_image2      AS image1, \n' +
    '       product.product_name        AS productName, \n' +
    '       product.product_description AS productDescription, \n' +
    '       orders.price                AS price, \n' +
    '       orders.shippind_date        AS shippingDate, \n' +
    '       orders.shipping_status      AS shippingStatus, \n' +
    '       orders.address              AS shippingAddress, \n' +
    '       orders.quantity             AS quantity \n' +
    'FROM   product \n' +
    '       INNER JOIN orders \n' +
    '               ON orders.customer_id = undefined\n' +
    '    '
}
